### DISCUSSION TITLE
🦊 Version 9.1 Beta for Firefox released

### DISCUSSION THREAD

**Discussion by paulrouget** [aclap-dev/video-downloadhelper#1071] [Announcements]
Version: v9.1 - for Firefox only, on the beta channel: https://www.downloadhelper.net/beta

What's new:

### New feature: Download History 

<img width="386" alt="Screenshot 2024-07-26 at 14 25 54" src="https://github.com/user-attachments/assets/68e3cc23-47c6-4d47-b6f7-dc6f40ac9598">

Give it a try, and tell us how you like it :)

### Fixed

- many many locales updated! Thanks to all the contributors :)
- added a History feature
- fixed queue logic for `Download As` actions
- fixed non-functional smartnaming
- improved video detection for some websites
- fixed slow popup for some websites
- fixed download title for some websites

**Comment by Mattmjs** [comment]
This beta hasn't fixed the download failure for nutror on  #1048 
It works directly using ffmpeg itself when using m3u8 link : https://playlist-videozz.k8s.eduzz.com/?t=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJrZXkiOiJiOTUyMzc4OS1mMTVlLTRjM2MtOGI5OS1kNmEzYzE2OWI3YmUiLCJzZXNzaW9uS2V5IjoiOWV3ZWNrZGwiLCJwcm9kdWNlcktleSI6IjI3NzU2OTQ5IiwibWF4UmVzb2x1dGlvbiI6MTA4MCwicHJvZHVjZXJFbWFpbCI6ImxpcGxhY2lkbzFAZ21haWwuY29tIiwiaW1wcmludCI6dHJ1ZSwiY29uc3VtZXJFbWFpbCI6InRwY29ycmFkaUBob3RtYWlsLmNvbSIsInN0b3JhZ2UiOiJyMiIsImlhdCI6MTcyMTk4Nzk3NH0.hASrXlz4RYz9TimC6KOASr3J3BreRzllgge5MEFK9JE&.m3u8

**Reply by paulrouget** [reply]
https://github.com/aclap-dev/video-downloadhelper/discussions/1048#discussioncomment-10198952

**Comment by Mattmjs** [comment]
You've solved the smartnaming problem , for example the brighteon website. It saves the name on the page.
I thought to try a download which launches download without dialog and also saves the name specified.

**Comment by mike156-ru** [comment]
The beta don't work on GetCource site, error in screenshot
![Снимок](https://github.com/user-attachments/assets/6a9bb824-9e57-463a-b75d-3acd628f7ba7)

**Reply by paulrouget** [reply]
1. Did it work before?
2. I need a URL to test

**Reply by paulrouget** [reply]
When you start a GetCourse download, once it fails, can you click on the Report button? I'll then take a look at your report.

**Reply by mike156-ru** [reply]
> When you start a GetCourse download, once it fails, can you click on the Report button? I'll then take a look at your report

Sorry, I can't find any button Report.  Where is it?

**Comment by nidan2** [comment]
[removed by moderator, comment against [our code of conduct](https://github.com/aclap-dev/video-downloadhelper/wiki/Code-Of-Conduct)]

**Reply by Mattmjs** [reply]
Of course there is a way to get a specific quality, you probably haven't taken the time to learn how to use it.
But if you don't like it, there is the option to switch back to the old legacy interface. Look for it in the settings.

**Comment by thelwyn** [comment]
Hi,

It doesn't detect any media on Youtube for me anymore, I'm not 100% sure if it started with the last update but the timing is very close. No problem on some other sites I've tried.

Version
9.1.0.41
Last update:
5th of  august 2024

I use it on Firefox 128.0.3

Exemple this page: https://www.youtube.com/watch?v=FtDZj42WPjg , i doesn't detect any media even after starting to play it.

I've tried to disable and re-enable the extension, same result.

Should I reinstall it? 

Edit: Sorry just saw this discussion afterward: https://github.com/aclap-dev/video-downloadhelper/discussions/1074
I'm in China and using a VPN to connect to Youtube, usually on a server set in France. I just tried and switched to a server in Switzerland, and that works... I feel discriminated ^^

**Reply by GrampaWildWilly** [reply]
There has been a suspicion circulating here that YouTube is rolling out some kind of change that will hurt VDH.  It's been a week now & I have yet to see a problem.  I tried your French mixtape video & VDH downloaded it just fine.  But I am in the USA so I don't need a VPN to get the content.  I just wanted to add to the data points about YouTube failures.  It's been at least a week since the suspicion was first raised by @paulrouget Paul that YouTube was rolling out something new.  I don't think that's the case.  I think the individual users are just doing something wrong.   I'm glad you were able to fix your problem via a VPN.

**Reply by thelwyn** [reply]
I don't think I'm doing anything wrong by connecting from where I live ^^
I found the workaround with the VPN location only because Paul mentioned in the other discussion that it seemed somehow location-related, few days before it was working fine.
